client/layout/controller.js

```diff
@@ -23,42 +23,48 @@ import DevDocs from 'devdocs';
 const getPages = () => {
 	const pages = [
 		{
-			container: Dashboard,
-			path: '/',
+			container: DevDocs,
```

Comment:
Might be nice to put DevDocs behind a feature flag that is only present in `development` mode.

Reply:
Done in https://github.com/woocommerce/wc-admin/pull/1500/commits/81607dc415c90a146867a373fdf157b892ff2631.